src/main/java/org/gridsuite/network/map/dto/mapper/ThreeWindingsTransformerInfosMapper.java

```diff
@@ -71,9 +71,7 @@ private static ThreeWindingsTransformerTabInfos toTabInfos(Identifiable<?> ident
                 .nominalVoltage1(terminal1.getVoltageLevel().getNominalV())
                 .nominalVoltage2(terminal2.getVoltageLevel().getNominalV())
                 .nominalVoltage3(terminal3.getVoltageLevel().getNominalV())
-                .country1(mapCountry(terminal1.getVoltageLevel().getSubstation().orElse(null)))
-                .country2(mapCountry(terminal2.getVoltageLevel().getSubstation().orElse(null)))
-                .country3(mapCountry(terminal3.getVoltageLevel().getSubstation().orElse(null)));
+                .country(mapCountry(terminal1.getVoltageLevel().getSubstation().orElse(null)));
```

Comment:
Maybe, instead of always get the 'terminal1' voltage level, it would be better to find the first terminal where we can get a substation ... because terminal1 voltage level may not be in a substation ...

Same remark for 2WT

Reply:
Done